src/MockClassifier.Api/Controllers/InstitutionController.cs

```diff
@@ -15,20 +28,30 @@ public class InstitutionController : ControllerBase
         [HttpPost]
         public AcceptedResult Post([FromBody] MessagesInput messages)
         {
-            var minsitries = new List<string>();
-
-            // TO DO
-            // Enumerate each string in messages and check for
-            // 1) Does the string match one of the pre-defined phrases? If so, work out the corresponding ministry and put it into minsitries
-            // 2) Does the string include one of more of the pre-defined tokens? If so, get the corresponding minsitries and put them into minsitries
-
-            foreach (var minsitry in minsitries)
+            foreach( var message in messages.Messages)
             {
-                // TO DO
-                // Invoke the DMR call back service to put a message on DMR for each misnitry
-            }
+                List<string> ministries = _naturalLanguageService.Classify(message).ToList();
+                ministries = ministries.Concat(_tokenService.Classify(message).ToList()).ToList();
```

Comment:
Maybe this is absolutely wild, but I would do something like this here:

```suggestion
List<string> ministries = ministries.Concat(_naturalLanguageService.Classify(message).ToList(), _tokenService.Classify(message).ToList());
```

Would this work?

Reply:
Actually no. It complains 'No overload for method 'Concat' takes 2 arguments'.

Reply:
Alright, I probably read the docs for the wrong thing then

Reply:
I would suggest using the following: 
```suggestion
ministries.AddRange(_tokenService.Classify(message).ToList());
```